devops/src/ax/devops/axsys/axsys_client.py

```diff
@@ -569,3 +569,7 @@ def delete_ingress(self, dnsname):
         """
         logger.info('Delete ingress policy for endpoint: %s', dnsname)
         return self.axmon_client.delete(INGRESS_METHOD + '/' + dnsname, value_only=True)
+
```

Comment:
you should not need this function... see comment below about using secrets in init container